libjupedsim/include/jupedsim/jupedsim.h

```diff
@@ -397,6 +397,39 @@ JPS_GeometryBuilder_Build(JPS_GeometryBuilder handle, JPS_ErrorMessage* errorMes
  */
 JUPEDSIM_API void JPS_GeometryBuilder_Free(JPS_GeometryBuilder handle);
 
+/**
+ * Opaque type for transition, describing route based decisions.
+ */
+typedef struct JPS_Transition_t* JPS_Transition;
+
+/**
+ * Create a fixed transition to stage
+ * @param[out] errorMessage if not NULL: will be set to a JPS_ErrorMessage in case of an error.
+ * @return Fixed transition to stage
```

Comment:
Need to mention that `NULL` will be returned on error.

Reply:
Added remark